File: assignment-1/python/main.py
from libs import Json, Yaml, Xml, Csv, Txt
from pprint import pprint

json_user_string = Json.parse("user.json");
yaml_user_string = Yaml.parse("user.yaml");
csv_user_string = Csv.parse("user.csv");
txt_user_string = Txt.parse("user.txt")
xml_user_string = Xml.parse("user.xml");

print("___________JSON___________")
pprint(json_user_string, width=80)
print("___________YAML___________")
pprint(yaml_user_string)
print("___________TXT____________")
pprint(txt_user_string)
print("___________XML____________")
pprint(xml_user_string["root"])
print("___________CSV____________")
pprint(csv_user_string)

File: assignment-1/typescript/src/main.ts
import { fileToString } from "./utils/file-reader";
import {
	parseCsv,
	parseJson,
	parseTxt,
	parseYaml,
	parseXml,
} from "./utils/parsers";


const userJson = parseJson(fileToString("user.json"));
const userYaml = parseYaml(fileToString("user.yaml"));
const userTxt =  parseTxt(fileToString("user.txt"))
const userCsv =  parseCsv(fileToString("user.csv"));
const userXml =  parseXml(fileToString("user.xml"));

console.log("___________JSON___________");
console.log(userJson);
console.log("___________YAML___________");
console.log(userYaml);
console.log("___________TXT___________");
console.log(userTxt);
console.log("___________CSV___________");
console.log(userCsv);
console.log("___________XML___________");
console.log("%o", userXml)
